composer.json

```diff
@@ -11,12 +11,13 @@
         }
     ],
     "require": {
-        "symfony/serializer": "^2.8|^3.0",
+        "php": "^7.0",
```

Comment:
The PHP version is not a hard requirement for this library, could you please remove this line?

Reply:
Done!